### PR TITLE
Fix types order

### LINKED ISSUE #714: Enforce our preferred types order in type switch
Implement before including linter #654.

### PR DESCRIPTION
**Pull request by fenogentov**
Refs #527.

### CODE REVIEW

File: internal/handlers/pg/msg_getlog.go [hunk 1]
```diff
@@ -85,6 +85,5 @@ func (h *Handler) MsgGetLog(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 	if err != nil {
 		return nil, lazyerrors.Error(err)
 	}
-
```
Comment (AlekSi) [inline internal/handlers/pg/msg_getlog.go:88]: All other commands have an empty line there, let's keep it

File: internal/handlers/common/projection.go [hunk 2]
```diff
@@ -142,7 +161,7 @@ func projectDocument(inclusion bool, doc *types.Document, projection *types.Docu
 				return err
 			}
 
-		case float64, int32, int64: // field: number
+		case float64: // field: number
```
Comment (AlekSi) [inline internal/handlers/common/projection.go:164]: Can we handle that case in our linter? I think if several types are in present in one `case`, they should be the in the right order, and the first type should be in the right place, but not second and following types

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: internal/handlers/common/projection.go (hunk 2 above)
Comment (fenogentov) [reply internal/handlers/common/projection.go]: linter has been submitted, now such cases are valid

File: internal/handlers/pg/msg_getlog.go (hunk 1 above)
Comment (fenogentov) [reply internal/handlers/pg/msg_getlog.go]: fix

**Comment by fenogentov** [comment]
code refactored #654.

**Pull request closed without merging**